cmds/id/id.go

```diff
@@ -0,0 +1,265 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// Print process information.
+//
+// Synopsis:
+//      id [-gGnu]
+//
+// Description:
+//      id displays the uid, guid and groups of the calling process
+// Options:
+//  		-g, --group     print only the effective group ID
+//		  -G, --groups    print all group IDs
+//		  -n, --name      print a name instead of a number, for -ugG
+//		  -u, --user      print only the effective user ID
+//
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"syscall"
+)
+
+var (
+	GROUP_FILE  = "/etc/group"
```

Comment:
don't use all caps and underlines. These can be const too
const (
groupFile = "/etc/group"
passwdFile = "/etc/passwd"

consider dropping the File part if there are no conflicts.